examples/ads130e08/default/README.md

```diff
@@ -0,0 +1,7 @@
+# Example for `ADS130E08` driver
+
+The datasheet can be found [here](https://www.ti.com/lit/ds/symlink/ads130e08.pdf?ts=1666137782883).
+
+## What it does
+
+The example configures one `ADS130E08` device on a `SPI` bus.
```

Comment:
would you elaborate a bit more? logs from a successful test would be also nice.

Reply:
Sure. I'll add it shortly.